index.js

```diff
@@ -84,7 +70,7 @@ module.exports = {
     "function-url-no-scheme-relative": true,
     "function-url-quotes": "always",
     "function-whitespace-after": "always",
-    "indentation": 2,
+    indentation: 2,
```

Comment:
Doesn't matter really as this is a JS object. There is no need to include quotation if the property name is just one word, i.e. prop name is not dynamic or include `-` or similar operator looking characters.

Reply:
Doesn't matter really as this is a JS object and it would be more consistent like this when we are not including the quotation marks. There is no need to include quotation if the property name is just one word, i.e. prop name is not dynamic or include `-` or similar operator looking characters.